templates/web/pages/bmviii_apply.tmpl

```diff
@@ -0,0 +1,431 @@
+{{ define "bmviii-apply" }}
+{{ template "head" . }}
+
+{{ template "bmvii navbar" . }}
+
+{{ template "flashes" .Flashes }}
+
+<section class="section bmvii-form" style='background-image: url({{ static_path "bmviii_application_background.svg" }}); background-color: #FFE4C3'>
```

Comment:
This background color doesn't match the background color of the actual background image:
![Screen Shot 2020-10-20 at 12 36 23](https://user-images.githubusercontent.com/10781907/96616864-7c1d8280-12c8-11eb-9e29-7bbe529770a8.png)

Reply:
oof good catch ty

Reply:
Switching it to an svg should resolve the issue